db/config.js

```diff
@@ -2,7 +2,9 @@ const path = require('path');
 
 try {
   require('dotenv').load({ path: path.resolve(__dirname, '..', '.env') });
-} catch (e) { }
```

Comment:
@Pinpickle Wrote this code, it might be worth asking him why this was ignored.

Reply:
In the past, `dotenv` would throw if `.env` didn't exist (which it doesn't in prod). I don't think this is the case anymore, but I'd give a quick check in a project with no `.env` file.

Reply:
You're right, I upgraded dotenv and it no longer throws in a project with no `.env` file.  I've removed the try-catch blocks.